examples/android/app/src/main/java/com/example/metriport/MainActivity.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.metriport
+
+import android.os.Bundle
+import android.webkit.WebViewClient
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_main.*
+
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        // WebViewClient allows you to handle
+        // onPageFinished and override Url loading.
+        webView.webViewClient = WebViewClient()
+
+        // this will load the url of the website
+        webView.loadUrl("http://192.168.0.138:3001/?token=1bEFTi9Zt_ljmG88pHj4U&colorMode=dark")
```

Comment:
Should we just set this to `https://connect.metriport.com/?token=demo` and make a comment saying that the token will need to be set with one our customer generates, and link to this section in our docs? https://docs.metriport.com/getting-started/connect-quickstart#4-link-to-the-metriport-connect-widget-in-your-app